common/app/views/commercial/cards/itemCard.scala.html

```diff
@@ -16,19 +16,19 @@
 <div class="@CssClassBuilder.linkFromStandardCard(item, optAdvertClassNames, optClassNames, useCardBranding)">
     <a href="@item.targetUrl"
        data-link-name="@omnitureId"
-       class="advert">
+       class="advert__container">
```

Comment:
In fact, I think the whole IE11 bug is caused by a selector in [_adverts.scss line 204](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/commercial/_adverts.scss#L204) that I wrote 😭  Instead, it should be: `.advert__row .advert__image-container`, not `.advert__row > * > .advert__image-container`